src/neptune/common/hardware/gpu/gpu_monitor.py

```diff
@@ -43,6 +45,22 @@ def get_card_usage_percent(self, card_index):
     def get_card_used_memory_in_bytes(self, card_index):
         return self.__nvml_get_or_else(lambda: nvmlDeviceGetMemoryInfo(nvmlDeviceGetHandleByIndex(card_index)).used)
 
+    def get_card_power_usage(self, card_index):
+        return self.__nvml_get_or_else(lambda: nvmlDeviceGetPowerUsage(nvmlDeviceGetHandleByIndex(card_index)))
+
+    def get_card_max_power_rating(self):
```

Comment:
Unused?

Reply:
Nope. Used in [`SystemResourceInfoFactory.__create_whole_system_resource_info()`](https://github.com/neptune-ai/neptune-client/blob/6bb4a9dc2be8a41a03e4e0e3c84db4370be9278c/src/neptune/common/hardware/resources/system_resource_info_factory.py#L46) and [`SystemResourceInfoFactory.__create_whole_system_resource_info()`](https://github.com/neptune-ai/neptune-client/blob/6bb4a9dc2be8a41a03e4e0e3c84db4370be9278c/src/neptune/common/hardware/resources/system_resource_info_factory.py#L57)

Reply:
Ok, thanks